requirements.txt

```diff
@@ -3,7 +3,7 @@ pep8
 oslo.utils
 shade>=1.7.0
 ansible>=2.1.0.0,<3,<=2.1.3
-python-novaclient
+python-novaclient<8.0.0
```

Comment:
This is only change we would really need to make 4.0.0.0r work. I will submit new PR for this